tests/embind/embind_benchmark.cpp

```diff
@@ -149,7 +149,7 @@ class Interface
     virtual void call0() = 0;
     virtual std::wstring call1(const std::wstring& str1, const std::wstring& str2) = 0;
     virtual void call_with_typed_array(size_t size, const void*) = 0;
-    virtual void call_with_memory_view(size_t size, const void*) = 0;
+    virtual void call_with_memory_view(size_t size, const unsigned int*) = 0;
```

Comment:
Why this change?

Reply:
I was getting the compile error
```
emscripten/wire.h:402:18: error: invalid application of 'sizeof' to an incomplete type 'void'
```
I'm a bit confused how this ever worked.

Reply:
But the line above was fine?  Strange.

Reply:
Yeah, `sizeof(void)` errors, but `sizeof(int)` is okay. `sizeof(void*)` is also okay, but not applied in this case.